WordPress/Tests/Theme/SearchFormUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+
+include( get_template_directory() . '/searchform.php' ); // Error.
+include( get_template_directory() . "/searchform.php" ); // Error.
+// searchform.php OK
```

Comment:
Please add some more examples of things which should be ok and which shouldn't. The current examples all use include while that is not what the sniff is sniffing for.

**NB: Pseudo code**:
```php
$path = get_template_directory() . '/searchform.php' ); // Error.
include $path;

$path = get_childtheme_directory() . "/searchform.php" ); // Error.
$path = get_childtheme_directory() . "/my-completely-different-searchform.php" ); // OK (will currently throw an error though).

$dummy_text_on_example_page = ".... so if you want to add an extra search form, just use `include searchform.php` somewhere in the template code..."; // What to do about things like this ? Should the sniff trigger on it or not ?
```